system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -810,8 +840,10 @@ construct_runtime!(
 		XcmpQueue: cumulus_pallet_xcmp_queue = 30,
 		PolkadotXcm: pallet_xcm = 31,
 		CumulusXcm: cumulus_pallet_xcm = 32,
+		// Temporary to migrate the remaining DMP messages:
```

Comment:
We should probably have some issue to remove them in the next upgrade.

Reply:
good point, [issue](https://github.com/polkadot-fellows/runtimes/issues/186) created and TODOs updated

https://github.com/polkadot-fellows/runtimes/pull/137/commits/97f81f9b9d759bbb5f7f64431ba71fa3718b2348